arangod/RocksDBEngine/RocksDBVPackIndex.h

```diff
@@ -147,6 +147,10 @@ class RocksDBVPackIndex : public RocksDBIndex {
   Result checkReplace(transaction::Methods& trx, RocksDBMethods* methods,
                       LocalDocumentId const& documentId, velocypack::Slice doc,
                       OperationOptions const& options) override;
+                      
+  Result checkOperation(transaction::Methods& trx, RocksDBMethods* methods,
```

Comment:
Perhaps `[[nodiscard]]` here too? Would the caller ignoring the returned `Result` be acceptable?